### PR TITLE
Add script to clean SPECTER temp files

### PR DESCRIPTION
**Pull request by haroldrubio**
This PR adds a simple script that clears the temp directory (as determined by Python) of sub-directories that are used by SPECTER. Specifically, when the `load_archive` function is called from the AllenNLP library, it creates a copy of the SPECTER model.

This script will not be necessary once AllenNLP gets updated, which automatically cleans up the temp files immediately.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,6 +153,11 @@ MFR_CHECKPOINT_DIR = '../expertise-utils/multifacet_recommender/mfr_model_checkp
 
 See `/expertise/service/README.md` for documentation on the API design and endpoints.
 
+When SPECTER is run as a service, it creates ~1GB of extra files get stored in a temporary directory and are not automatically cleaned up. In order to cleanup these files, you can run the following command. **WARNING**: Any currently running jobs have the potential to fail when performing this operation
+```
+python -m expertise.clean_tmp
```
Comment (melisabok) [inline README.md:158]: Should we call it like this from the CRON job?

**Comment by purujitgoyal** [comment]
Rather than using a python script, we can directly use a simple bash script, like this:

`find /tmp/* -prune -type d -mtime +1 -exec rm -r {} \;`

This will remove all the directories in `tmp` folder which were created more than a day ago.

**Comment by melisabok** [comment]
@haroldrubio do we need this to be installed before TMLR?

File: clean_tmp.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+find /tmp/* -prune -type d -mtime +0.084 -exec rm -r {} \;
```
Comment (purujitgoyal) [inline clean_tmp.sh:1]: @haroldrubio do you know is there a particular pattern to directory names which get created? I'm worried removing everything from `/tmp` might cause problems elsewhere

Comment (haroldrubio) [reply clean_tmp.sh:1]: Yup the directories start with `tmp`, so `/tmp/tmp71atajqf`. Is would `/tmp/tmp*` be the correct regex?

Comment (purujitgoyal) [reply clean_tmp.sh:1]: I believe so.

**Review by purujitgoyal — APPROVED** [review]

**Review by melisabok — APPROVED** [review]

**Pull request merged**